app/models/domain/StringFieldRegex.scala

```diff
@@ -28,4 +28,6 @@ object StringFieldRegex {
   val eoriNumberRegex: Regex       = "^(?i)(gb|xi)[0-9]*$".r
 
   val tirIdNumberRegex: Regex = "^[a-zA-Z]{3}\\/[0-9]{3}\\/[0-9]*$".r
+
+  val postalCodeRegex: Regex = "^[a-zA-Z\\s*0-9]*$".r
```

Comment:
This regex will cover for invalid characters but should we add one in for the format at this point too? I think there's one in P5 Arrivals